packages/linkChecker/checkExternalLinks.ts

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env node
+import fs from "node:fs/promises";
+import path from "node:path";
+import { glob } from "glob";
```

Comment:
I like that this is in a separate package now, but shouldn't there be a package.json to declare the depedency on `glob` and typescript?  Maybe check the i18n package.json for reference?